fontbe/src/font.rs

```diff
@@ -28,20 +28,20 @@ enum TableType {
     Variable,
 }
 
-const TABLES_TO_MERGE: &[(WorkId, Tag, TableType)] = &[
-    (WorkId::Avar, Avar::TAG, TableType::Variable),
-    (WorkId::Cmap, Cmap::TAG, TableType::Static),
-    (WorkId::Fvar, Fvar::TAG, TableType::Variable),
-    (WorkId::Head, Head::TAG, TableType::Static),
-    (WorkId::Hhea, Hhea::TAG, TableType::Static),
-    (WorkId::Hmtx, Hmtx::TAG, TableType::Static),
-    (WorkId::Glyf, Glyf::TAG, TableType::Static),
-    (WorkId::Gvar, Gvar::TAG, TableType::Variable),
-    (WorkId::Loca, Loca::TAG, TableType::Static),
-    (WorkId::Maxp, Maxp::TAG, TableType::Static),
-    (WorkId::Name, Name::TAG, TableType::Static),
-    (WorkId::Os2, Os2::TAG, TableType::Static),
-    (WorkId::Post, Post::TAG, TableType::Static),
+const TABLES_TO_MERGE: &[(WorkId, Tag, TableType, usize)] = &[
```

Comment:
This deserves a comment/apology

Reply:
I agree. Fixed by shoving loca format into it's own slot instead of being sneaky.